app/controllers/api/v1/users_controller.rb

```diff
@@ -3,46 +3,40 @@ module V1
     class UsersController  < ApplicationController
       skip_before_action :verify_authenticity_token, if: :json_request?
       skip_before_filter  :verify_authenticity_token
+      before_action :load_user, only: [:show, :update, :destroy]
 
 
-    def create
-      user = User.create(user_params)
-      if user.save
-        render json: { message: t("user.signup.success") }, status: :created
-      else
-        render json: { error: user.errors }, status: :not_acceptable
+      def create
+        user = User.create(user_params)
+        if user.save
+          render json: { message: t("user.signup.success") }, status: :created
+        else
+          render json: { error: user.errors }, status: :not_acceptable
+        end
       end
-    end
 
       def update
-        user = User.find(params[:id])
-        if user.update_attributes(user_params)
+        if @user.update_attributes(user_params)
           render json: { message: t("user.update.success") }, status: :ok
         else
-          render json: { error: user.errors }, status: :not_acceptable
+          render json: { error: @user.errors }, status: :not_acceptable
         end
       end
 
       def show
-        user = User.find(params[:id])
-        if user
-          data = {
-            'first_name' => user.first_name,
-            'last_name' => user.last_name,
-            'email' => user.email,
-            'avatar' => user.avatar
-          }
-          render json: { data: data }, status: :ok
+        if @user
+          render 'users.json.jbuilder', status: :ok
```

Comment:
No need to explicitly specify render method, rewrite this to, 
render json: { error: 'No Content Available'}, status: :no_content if @user.empty?